code/game/objects/items/weapons/defib.dm

```diff
@@ -372,8 +372,9 @@
 	H.apply_damage(burn_damage_amt, BURN, BP_CHEST)
 
 	var/obj/item/organ/internal/heart/heart = H.internal_organs_by_name[BP_HEART]
-	heart.scp3349_induced = FALSE	// part of SCP-3349. electric shocks temporarily stop effect
-	to_chat(H, SPAN_INFO("The euphoric sensation ends."))
+	if(heart.SCP.designation == "3349-1")	// part of SCP-3349. electric shocks temporarily stop effect
+		SCP.Destroy()
```

Comment:
????????

Reply:
This was a way of checking for the SCP instance existing without using a specialized var in the heart organ.

Reply:
Err I think at the time I wasn't sure how to handle nulling a datum and the datum's destroy proc handles getting everything done neatly

Lmk how it should be done